### PR TITLE
Fix OPA logging structure and target

### PR DESCRIPTION
**Pull request by amirbenun**
Fixes https://github.com/elastic/security-team/issues/4168

### CODE REVIEW

**Comment by amirbenun** [comment]
With the change:
image

**Comment by oren-zohar** [comment]
@amirbenun the logs are still parsed incorrectly, is this PR could also handle it?

**Comment by amirbenun** [comment]
Yes, right now the log message is still escaped but the default is to have no logs at all.
The logs are escaped since they are printed to the stdout instead of to the dedicated file.
In order to fix that we will have to implement OPA's `logging.Logger` interface in a way that delegates to the beats logger.
Do we want to have it right away or should I create a ticket? What do you think? @oren-zohar @yashtewari

**Comment by oren-zohar** [comment]
> Yes, right now the log message is still escaped but the default is to have no logs at all.
> The logs are escaped since they are printed to the stdout instead of to the dedicated file.
> In order to fix that we will have to implement OPA's `logging.Logger` interface in a way that delegates to the beats logger.
> Do we want to have it right away or should I create a ticket? What do you think? @oren-zohar @yashtewari

@amirbenun If you think it's a quick fix we can do it here then we can update the current ticket, otherwise opening a separate ticket sounds good

File: _meta/config/cloudbeat.common.yml.tmpl [hunk 1]
```diff
@@ -47,4 +47,6 @@ cloudbeat:
         patterns: [
           "/hostfs/etc/kubernetes/kubelet/kubelet-config.json",
           "/hostfs/var/lib/kubelet/kubeconfig",
-        ]
\ No newline at end of file
+        ]
+  evaluator:
```
Comment (DaveSys911) [inline _meta/config/cloudbeat.common.yml.tmpl:51]: Please add it to cloudbeat.common.yml.tmpl & be sure to run `mage check`

**Review by DaveSys911 — APPROVED** [review]
See comment about adding change to cloudbeat reference tmpl

File: config/config_test.go [hunk 2]
```diff
@@ -133,237 +133,6 @@ func (s *ConfigTestSuite) TestRuntimeCfgExists() {
 	}
 }
 
-func (s *ConfigTestSuite) TestConfigUpdate() {
```
Comment (oren-zohar) [inline config/config_test.go:136]: What is the reason for removing the tests here?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: config/config_test.go (hunk 2 above)
Comment (amirbenun) [reply config/config_test.go:136]: The `Update` function is not being used anymore

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**